lib/src/date-input/Calendar.tsx

```diff
@@ -35,19 +35,23 @@ const getDays = (innerDate: Dayjs) => {
   return monthDayCells;
 };
 
+const getDateToFocus = (selectedDate, innerDate, today) => {
+  return selectedDate?.get("month") === innerDate.get("month") && selectedDate?.get("year") === innerDate.get("year")
+    ? selectedDate
+    : today.get("month") === innerDate.get("month") && today.get("year") === innerDate.get("year")
+    ? today
+    : innerDate.set("date", 1);
+};
+
 const isDaySelected = (date: { day: number; month: number; year: number }, selectedDate) =>
   selectedDate?.get("month") === date.month &&
   selectedDate?.get("year") === date.year &&
   selectedDate?.get("date") === date.day;
 
 const Calendar = ({ selectedDate, innerDate, onInnerDateChange, onDaySelect }: CalendarPropsType): JSX.Element => {
-  const [dateToFocus, setDateToFocus] = useState(
-    selectedDate?.get("month") === innerDate.get("month") && selectedDate?.get("year") === innerDate.get("year")
-      ? selectedDate
-      : dayjs()
-  );
+  const [today] = useState(dayjs());
```

Comment:
Why `today` variable is a stateful variable? It never modifies its value so it may be a better approach to create a global variable outside the component.

Not everything needs to be a stateful variable, try always to keep the component's state as light as possible.

Reply:
I agree, but in this case not being a stateful variable will create a new dayjs object for each render, and having it as a global variable was causing problems in the test, I will try to check it again in case I find a better solution.

Reply:
What I meant by global variable was outside of the component. It is rare that you have problems in the tests 🤔

Reply:
The issue lies in mocking the value of today in the tests. It seems unable to change today's value if it is instantiated outside the component.